### ISSUE TITLE
Looking for the SG ports

### ISSUE THREAD

**Issue by dirkjanfaber** [raomin/ESPAltherma#442]
First of all, what a superb project. I got connecting the ESP32 to my `EHVX08S18D6V` figured out by following the documentation. Thanks for developing this.

I would like to steer the device a bit too, mainly by enabling and disabling the SG ports, if possible. By looking at the labelling of the connections, I don't clearly see which ports that would be. On the right terminal I got the following connections:

<img width="1120" alt="image" src="https://github.com/user-attachments/assets/de011c4e-3033-40cf-80f5-3cde82bfc99b">

On the left terminal, the labels are a bit harder to read because of all the cables in front of it, but they are labeled as follows:
<img width="930" alt="image" src="https://github.com/user-attachments/assets/9dbc6951-7c3e-43a5-ab0e-39ce80c4a8d3">

Any idea what ports are linked to the SG ports?

**Comment by dirkjanfaber** [comment]
Of course it could also be one of the other ports on the board: X42A, X32A, X85A, X38A or X33A. All suggestions are welcome. 

![AADE9B99-85A5-4FC7-B6E6-B54EEF71F990_1_102_o](https://github.com/user-attachments/assets/4876c59c-ecf6-48c0-a9d5-b79063461b9b)

**Comment by dirkjanfaber** [comment]
After some googling, it looks like the `S1S` contact might be the _benefit kWh rate power supply contact_. Which, if I understand correctly, seems equivalent to the SG1 contact. 

Other things I read is that only the LAN adapter will provide the extra ports.

**Comment by luksa** [comment]
Yup, for older models, you need the [LAN adapter](https://www.daikin.eu/en_us/products/product.html/BRP069A61.html). But make sure to get the one with SmartGrid functionality.

**Comment by raomin** [comment]
Cleaning up old resolved issues, this was answered in the thread.
